audiomentations/augmentations/transforms.py

```diff
@@ -1692,6 +1724,7 @@ def __init__(
             Must be a multiple of 6
         :param max_rolloff: Maximum filter roll-off (in db/octave)
             Must be a multiple of 6
+        :param zero_phase: Whether filtering should be zero phase
```

Comment:
It would be nice to have a sentence of two that explain the implications of this in a way most developers can understand.

E.g. will audio quality improve (audibly or just in theory)? Will it be slower? By how much?